study-builder/studies/pancan/emails/consent_completed.tmpl.html

```diff
@@ -0,0 +1,124 @@
+    }
+
+    @font-face {
+      font-family: 'Roboto', sans-serif;
+      src: url(https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700);
+      font-weight: 300;
+      font-style: normal;
+    }
+
+    @media screen and (max-width: 480px) {
+      body {
+        padding-left: 10px!important;
+      }
+
+      .table {
+        width: 90% !important;
+      }
+    }
+  </style>
+</head>
+
+<body style="background: #ffffff; width: 100%; height: 100%; padding-top: 60px; padding-left: 60px;">
+  <table style="width: 540px; padding: 0; -webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-family: Roboto, sans-serif; border-collapse: collapse !important;"
+         border="0" cellpadding="0" cellspacing="0" width="100%"
+         class="table">
+    <tr style="width: 540px;">
+      <td align="left" style="margin: 0; padding: 0; font-family: Roboto, sans-serif; width: 278px; height: 33px">
+        <a href="{{baseWebUrl}}" target="_blank" style="cursor: pointer; text-decoration: none;">
+          <img src="https://storage.googleapis.com/${assetsBucketName}/pancan/logo-count-me-in.png"
+               alt="${text['logo_alt']}" style="-ms-interpolation-mode: bicubic; border: 0; outline: none; text-decoration: none;"
+               width="230px" height="111px" border="0">
+        </a>
+      </td>
+    </tr>
+
+    <tr style="width: 540px;">
+      <td style="padding: 50px 0 0 0;">
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
+          {{\#if proxy_firstName}}${text["salutation_parent"]}{{else}}${text["salutation_self"]}{{/if}}
+        </p>
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">${text["thank_you"]}</p>
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">${text["instructions"]}</p>
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">${text["provide_samples"]}</p>
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">${text["complete_process"]}</p>
+      </td>
+    </tr>
+    <tr style="width: 540px;">
+      <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+        <div>&nbsp; <a href="{{baseWebUrl}}/dashboard" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```

Comment:
Not sure if the link correct for the button "Contact Information Form" ?

Reply:
For linking to activities, what we usually do is put a link like this:
```
{{baseWebUrl}}/activity-link/{{activityInstanceGuid}}
```
The backend will provide the `activityInstanceGuid` for the "Contact Information Form", and on the frontend there will be a component at path `/activity-link` that will take that path parameter and show the exact activity.

Reply:
Putting a link to the dashboard is also fine. User will see it listed in their dashboard and can open it themselves. Really just depends on what the study staff prefers.